src/js/view/controls/components/settings/menu.js

```diff
@@ -53,7 +53,7 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
             addDocumentListeners(documentClickHandler);
 
             if (isDefault) {
-                if (!window.event.pointerType) {
```

Comment:
We should pass the event (`evt`) into open and close from the methods calling them. The logic on line 38 to determine `isKeyEvent` or `isDefault` should be in these methods.

Reply:
Do not use `window.event`. It is not a reliable way to get event context for code that is part of an event hander chain.

Reply:
@karimJWP @hghazzi you guys should pass this to @johnBartos to address. The event handling needs quick fix for release.

Reply:
If we can't address padding click listener events into here than let's remove the check. What's wrong with always focusing? Is this only supposed to happen on key press? If so you can do that using 'keyup' in controls.js

Reply:
@robwalch  the reason we don't want to pass events is because of a comment left at https://github.com/jwplayer/jwplayer/blob/master/src/js/view/controls/components/button.js#L17 . It was left last year by Evol, do you think this is still relevant ?